### ISSUE TITLE
[BUG] Toggling verbose timestamp script kicks you from KES

### ISSUE THREAD

**Issue by McBinary** [aclist/kbin-kes#60]
**Describe the bug**
Toggling the verbose timestamp feature on and then back off force closes KES and reloads the browser window.

**To Reproduce**
Open KES, General > Verbose TimeStamps > toggle the switch on then off again

**Expected behavior**
The KES shade continues to stay open during the other toggles I've attempted.

**Screenshots**
n/a

**Megamod version:**
KES 1.0.3

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Brave
 - Browser version: 1.52.126

**Comment by McBinary** [comment]
I skipped over the part that says that it will reload... I stopped reading that line after the word "swedish"... My fault. Closing.

**Comment by aclist** [comment]
If you take a look at the description for that mod, it says:

> Use adapted ISO8601 (Swedish-like) date/time syntax. Note: toggling off will reload the page

This mod is somewhat unique from the other ones. Originally it worked the way you describe, but we noticed that kbin tries to update timestamps after a minute has passed (34 minutes ago, 35 minutes ago...), causing these values to be overwritten. So we have to watch for changes to timestamps and restyle them in the new format. 

The consequence of this is that reverting the timestamps triggers a large number of changes to all of the timestamp nodes above, which floods the observer used to watch for these time tick events and overwhelms the browser. In this case, we just set the feature to off and reload so that everything resets.

There is at least one other script that depends on a reload to reset, but we are trying to minimize these.
